### PR TITLE
Add CODEOWNERS, remove .pullapprove.yml

### PR DESCRIPTION
**Pull request by jdolitsky**

### CODE REVIEW

**Review by tianon — APPROVED** [review]

**Comment by tianon** [comment]
cc @opencontainers/runtime-tools-maintainers

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @crosbymichael @dqminh @hqhq @liangchenye @mashimiao @mrunalp @q384566678 @tianon @vbatts @vishh
```
Comment (hqhq) [inline CODEOWNERS:1]: Can we keep using `runtime-tools-maintainers` so we don't have to maintain multiple maintainer lists?

According to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax , I guest we can specify a team name here.

Comment (kolyshkin) [reply CODEOWNERS:1]: @jdolitsky PTAL ^^

Comment (vbatts) [reply CODEOWNERS:1]: I think making it a group name, and then just managing the group members is a solid choice

**Comment by kolyshkin** [comment]
Closing in favor of https://github.com/opencontainers/runtime-tools/pull/756

**Pull request closed without merging**